bokun.yaml

```diff
@@ -5537,413 +5734,535 @@ definitions:
+      languages:
+        type: array
+        items:
+          type: string
+        description: 'The list of languages which the product content is available in.'
+      paymentCurrencies:
+        type: array
+        description: 'Deprecated.'
+        items:
+          type: string
+      customFields:
+        type: array
+        description: 'In order to have custom fields for the product, they have to be enabled, in "Advanced settings->Custom input fields".'
+        items:
+          $ref: '#/definitions/CustomFieldDto'
+      tagGroups:
+        type: array
+        description: 'The list of different groups of custom parameters, created for to this activity. One example of tag groups - "Activity type". All tag groups present in system get be fetched by GET /tag.json/groups. '
+        items:
+          $ref: '#/definitions/TagGroupDto'
+      categories:
+        type: array
+        items:
+          $ref: '#/definitions/CategoryDto'
+      keyPhoto:
+        $ref: '#/definitions/PhotoDto'
+        description: 'The photo selected for use as the key photo for the product in lists and overviews.'
+      photos:
+        type: array
+        description: 'The list of photos showcasing this product.'
+        items:
+          $ref: '#/definitions/PhotoDto'
+      videos:
+        type: array
+        description: 'The list of videos showcasing the product.'
+        items:
+          $ref: '#/definitions/VideoDto'
+      vendor:
         $ref: '#/definitions/VendorDto'
-      actualId:
+        description: 'The vendor that operates this activity.'
+      boxedVendor:
+        $ref: '#/definitions/VendorDto'
+        description: 'If this is a boxed activity, then this field holds the ID of the vendor that owns the activity inside the box.'
+      storedExternally:
+        type: boolean
+      pluginId:
+        type: string
+        description: 'Deprecated.'
+      reviewRating:
+        type: number
+        format: double
+        description: 'Deprecated.'
+      reviewCount:
         type: integer
         format: int64
-      agendaItems:
+        description: 'Deprecated.'
+      activityType:
+        $ref: '#/definitions/ActivityType'
+        description: 'Specifies the type of the activity.'
+      bookingType:
+        $ref: '#/definitions/ActivityBookingType'
+        description: 'Specifies how the activity is booked: as a pass, date based, or date and time based.'
+      scheduleType:
+        $ref: '#/definitions/ActivityScheduleType'
+        description: 'Specifies how activity start times are scheduled. Only used if bookingType != PASS.'
+      capacityType:
+        $ref: '#/definitions/ActivityCapacityType'
+        description: 'Specifies whether capacity is limited, free sell, or on request.'
+      passExpiryType:
+        $ref: '#/definitions/PassExpiryType'
+        description: 'Only used if bookingType == PASS. Specifies how the pass expires after purchase. Could be'
```

Comment:
Could be what?

Reply:
"...non-empty, if this product had this value previously."
Fixed.